saleor/graphql/app/mutations/app_token_verify.py

```diff
@@ -25,8 +26,11 @@ class Meta:
 
     @classmethod
     def perform_mutation(cls, _root, _info, /, *, token: str):  # type: ignore[override]
+        apps = models.App.objects.filter(
+            is_active=True, removed_at__isnull=True
```

Comment:
Shouldn't we create manager function for `models.App.objects.filter(removed_at__isnull=True)` as it's used multiple times

Reply:
I don't think that wrapping one filter in a separate function brings value.

Reply:
You are using it multiple times and from my perspective it's easier to use `App.objects.active()` or `App.objects.not_removed()` instead of doing `models.App.objects.filter(removed_at__isnull=True)`

Reply:
https://github.com/saleor/saleor/issues/14681